### PR TITLE
Add instructions for Project 1

### PR DESCRIPTION
**Pull request by poojanilangekar**

### CODE REVIEW

File: project1/queries.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+queries[0] = """
+select airportid, city
+from airports
+order by city;
+"""
+
+### 1. Write a query to find the names of customers who have flights on a friday and 
+###    first name that has a second letter is not a vowel.
+###    Do not include the youngest customer that satisfies the above conditions in the results.
```
Comment (ctring) [inline project1/queries.py]: "Do not include the youngest customer **among those** that satisfies..."

File: project1/README.md [hunk 2]
```diff
@@ -14,9 +14,14 @@ The following assumes you have gone through PostgreSQL instructions from Project
 **Note:** We will test your queries on a different, larger dataset.
 
 ### Getting started
```
Comment (ctring) [inline project1/README.md:16]: typo in "You can restart the continer..."

File: project1/README.md [hunk 3]
```diff
@@ -14,9 +14,14 @@ The following assumes you have gone through PostgreSQL instructions from Project
...
-Similar to Project0, you will build the container with `docker build -t "cmsc424-project1" .` in the `project1` directory. Next you will need to start the container using `docker run -v $PWD:/home/project1 -ti -p 8888:8888 -p 5432:5432 cmsc424-project1:latest`. Alternately, if you are using your host machine to start the PostgreSQL server, follow the instructions mentioned in the [Project0 README](https://github.com/abadid/cmsc424-spring2023/tree/main/project0).
+Similar to Project0, you will build the container with `docker build -t "cmsc424-project1" .` in the `project1` directory. Next you will need to start the container using `docker run -v $PWD:/home/project1 -ti -p 8888:8888 -p 5432:5432 --name project1  cmsc424-project1:latest`. We have already created and loaded the flights database in the docker container. If you are using a native installation of postgres, will need to create and load the `flights` database using the `small.sql` file.  You can restart the continer with the following commands:
```
Comment (ctring) [inline project1/README.md]: typo in "You can restart the continer..."

File: project1/queries.py [hunk 4]
```diff
@@ -93,15 +94,19 @@
...
 ###    Return the name, start and end date of the customers flights.
 ### Output: customer_name, start_date, end_date
 ### Order: by customer_name
 queries[8] = """
 select 0;
 """
 
-### 9. Write a query to find all flights belonging to the same airline that had a layover in JFK between 1 and 3 hours in length (inclusive)
+### 9. A layover consists of set of two flights where the destination of the first flight is the same 
+###    as the source of the second flight. Additiobally, the arrival of the first flight must be before the
```
Comment (ctring) [inline project1/queries.py]: Typo in "Additionally"

File: project1/queries.py [hunk 5]
```diff
@@ -93,15 +94,19 @@
...
+###    departure of the first flight. 
+###    Write a query to find all flights belonging to the same airline that had a layover in JFK between
```
Comment (ctring) [inline project1/queries.py]: "... to find all **pairs of flights** belonging to ..."

**Review by ctring — APPROVED** [review]

File: project1/README.md (hunk 3 above)
Comment (poojanilangekar) [reply project1/README.md]: Done.

File: project1/queries.py (hunk 5 above)
Comment (poojanilangekar) [reply project1/queries.py]: Done.

File: project1/queries.py (hunk 4 above)
Comment (poojanilangekar) [reply project1/queries.py]: Done.

File: project1/queries.py (hunk 1 above)
Comment (poojanilangekar) [reply project1/queries.py]: Done.

**Review by abadid — APPROVED** [review]

**Pull request merged**